### DISCUSSION TITLE
Presto vs Find(All)Markers for differential expression

### DISCUSSION THREAD

**Discussion by edt711** [satijalab/seurat#4105] [Q&A] [answered]
Hello @satijalab,

Thanks for all of your wonderful work on Seurat! I see that in your WNN vignette, you use presto to determine cluster-specific gene enrichment. I assume that it can also be used for performing differential expression. However, is the analysis performed by presto better than the old FindMarkers (or FindAllMarkers) functions? Or is it just faster? 

Also, in the presto output, do you know if the logFC value is log2 fold change, or natural log (like the output of FindMarkers was in older versions of Seurat)? 

Thanks,
Eric

**Comment by timoast** [comment, marked as answer]
You can find more information about presto here: https://github.com/immunogenomics/presto

It is a faster way of computing a Wilcoxon rank sum test for differential expression

**Reply by edt711** [reply]
Hi Tim,

Yeah, I had checked their Github page, but I couldn't tell if there was any difference between it and the Wilcox test performed by FindMarkers, other than speed. If speed is the only difference, then we should be fine using FindMarkers for differential expression in Seurat v4/for multiomic datasets, no?

Thanks,
Eric

**Reply by timoast** [reply]
Speed is the only advantage, but it is substantially faster. From their GitHub readme:

>  Latest benchmark ran 1 million observations, 1K features, and 10 groups in 16 seconds (sparse input) and 85 seconds (dense input).

**Reply by edt711** [reply]
Perfect. Thanks again, Tim!
